src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../Icon';
+import { Popover } from '../Popover';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+
+/**
+ * FieldLevelHelp Component for Patternfly React
+ */
+const FieldLevelHelp = ({ children, contentType, content, ...props }) => {
+  const trigger = contentType === 'popover' ? 'click' : 'hover focus';
+  const htmlContent = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: content
+      }}
+    />
+  );
+  const overlay =
+    contentType === 'popover' ? (
+      <Popover id={contentType}>{htmlContent}</Popover>
+    ) : (
+      <Tooltip id={contentType}>{htmlContent}</Tooltip>
+    );
+  const rootClose = true;
+
+  return (
+    <OverlayTrigger
+      overlay={overlay}
+      placement={'top'}
+      trigger={trigger.split(' ')}
+      rootClose={rootClose}
+    >
+      <label>
+        {children + ' '}
+        <Icon type="pf" name={'info'} style={{ color: '#0088ce' }} />
```

Comment:
we shouldn't use an inline style here add the `btn-link` class instead.

Reply:
`btn-link` introduces an underline that we typically see for links on a mouse hover, as shown below -
<img width="100" alt="screen shot 2018-01-24 at 11 29 32 am" src="https://user-images.githubusercontent.com/1538216/35352819-00a28a34-00fa-11e8-8632-d286bba67c9a.png">

Is there a way to remove that?